charts/public-service/values.yaml

```diff
@@ -0,0 +1,48 @@
+image:
+  repository: 'MUST_BE_OVERRIDDEN'  # either 'sphereio/commercetools-payone-integration' or 'commercetoolsps/commercetools-paypal-plus-integration/'
+  tag: 'MUST_BE_OVERRIDDEN'  # set tag here
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+service:
+  type: ClusterIP
+  externalPort: 80
+  internalPort: 8080
+
+rollingUpdateStrategy:
+  maxSurge: 1
+  maxUnavailable: 0
+
+portName: backend-port
+
+## specifies the resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources: {}
+#  requests:
+#    cpu: 100m
+#    memory: 300Mi
+#  limits:
+#    cpu: 100m
+#    memory: 400Mi
+
+# non sensitive environment variables
+nonSensitiveEnvs: {}
+
+# sensitive environment variables
+sensitiveEnvs: {}
+
+## Ingress annotations
+##
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  # hosts:
+    # - service-name.local
```

Comment:
the host value should be uncommented here (default)

Reply:
b7e826b